doc/src/site/sphinx/ml/sw_glrm.rst

```diff
@@ -0,0 +1,105 @@
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        First, let's start Sparkling Shell as
+
+        .. code:: shell
+
+            ./bin/sparkling-shell
+
+        Start H2O cluster inside the Spark environment
+
+        .. code:: scala
+
+            import ai.h2o.sparkling._
+            import java.net.URI
+            val hc = H2OContext.getOrCreate()
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: scala
+
+	        import org.apache.spark.SparkFiles
+            spark.sparkContext.addFile("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/iris/iris_wheader.csv")
+	        val sparkDF = spark.read.option("header", "true").option("inferSchema", "true").csv(SparkFiles.get("iris_wheader.csv"))
+
+        Train the model. You can configure all the available GLRM arguments using provided setters.
+
+        .. code:: scala
+
+            import ai.h2o.sparkling.ml.algos.H2OGLRM
+            val estimator = new H2OGLRM()
+              .setK(3) // Number of output dimensions
+              .setLoss("quadratic")
+              .setGammaX(0.5)
+              .setGammaY(0.5)
+              .setTransform("standardize")
+            val model = estimator.fit(sparkDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the model as:
+
+        .. code:: scala
+
+            model.getModelDetails()
+
+        Transform your dataset, the output will be under the ``prediction`` column.
+
+        .. code:: scala
+
+            model.transform(sparkDF).show(false)
+
+
+    .. tab-container:: Python
+        :title: Python
+
+        First, let's start PySparkling Shell as
+
+        .. code:: shell
+
+            ./bin/pysparkling
+
+        Start H2O cluster inside the Spark environment
+
+        .. code:: python
+
+            from pysparkling import *
+            hc = H2OContext.getOrCreate()
+
+        Parse the data using H2O and convert them to Spark Frame
+
+        .. code:: python
+
+            import h2o
+            frame = h2o.import_file("https://raw.githubusercontent.com/h2oai/sparkling-water/master/examples/smalldata/iris/iris_wheader.csv")
+            sparkDF = hc.asSparkFrame(frame)
+
+        Train the model. You can configure all the available GLRM arguments using provided setters or constructor parameters.
+
+        .. code:: python
+
+            from pysparkling.ml import H2OGLRM
+            estimator = H2OGLRM(k=3, loss="quadratic", gammaX=0.5, gammaY=0.5, transform="standardize")
+            model = estimator.fit(sparkDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the model as:
+
+        .. code:: python
+
+            model.getModelDetails()
+
+        Transform your dataset, the output will be under the ``prediction`` column.
+
+        .. code:: python
+
+            model.transform(sparkDF).show(truncate = False)
```

Comment:
It would be good to mention how to obtain the reduced frame.

We expose parameter `representation_name` which is the key of the frame where the reduced frame gets saved as H2OFrame.

WDYT?

Reply:
will do thanks!